src/components/bookappointment/payment/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from 'react';
+import { loadStripe } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+import CheckoutForm from '../../checkoutform';
+import axiosInstance from '../../../utils/apis/axios';
+
+const stripePromise = loadStripe('pk_test_51NGyZkEFeMbkGJMBieYj5kVZnmRPzkZ5JVEe6rd4IvSOu1XAAxmN2arJ0LJMGKxFr6LNdyemP9q5tixH3RPUu3Vl00U8U0DSmt');
+
+const Payment = () => {
+  const [clientSecret, setClientSecret] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {

```

Comment:
```suggestion
    const getClientSecret = async () => {

```
For sure axios request going to fetch data, But what this request about!
Use meaningful var's naming